package.json

```diff
@@ -94,6 +94,9 @@
     "tmp": "^0.0.33",
     "what-input": "^5.1.2"
   },
+  "resolutions": {
```

Comment:
it is better to put this section after dependencies sections (i.e. `dependencies`, `devDependencies`, `peerDependencies`) - that's the general practice

Reply:
will change

Reply:
done